packages/p2p/src/components/buy-sell/buy-sell-row.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table, Text, Button, Icon } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { observer } from 'mobx-react-lite';
+import { Localize, localize } from 'Components/i18next';
+import UserAvatar from 'Components/user/user-avatar';
+import { useStores } from 'Stores';
+import './buy-sell-row.scss';
+
+const BuySellRow = observer(({ row: advert, setSelectedAdvert, showAdvertiserPage }) => {
+    if (advert.id === 'WATCH_THIS_SPACE') {
+        // This allows for the sliding animation on the Buy/Sell toggle as it pushes
+        // an empty item with an item that holds the same height of the toggle container.
+        // Also see: buy-sell-table.jsx
+        return <div style={{ height: '77px' }} />;
+    }
+
+    const { general_store } = useStores();
+    const {
+        account_currency,
+        counterparty_type,
+        local_currency,
+        max_order_amount_limit_display,
+        min_order_amount_limit_display,
+        price_display,
+    } = advert;
+
+    const is_my_advert = advert.advertiser_details.id === general_store.advertiser_id;
+    const is_buy_advert = counterparty_type === 'buy';
```

Comment:
Please use constant from `packages/p2p/src/constants/buy-sell.js`.

Reply:
This is done.